apps/src/gamelab/GameLabGame.js

```diff
@@ -37,4 +40,13 @@ Object.defineProperty(GameLabGame.prototype, 'frameRate', {
   }
 });
 
+Object.defineProperty(GameLabGame.prototype, 'seconds', {
+  enumerable: true,
+  get: function () {
+    const currentDate = new Date();
+    const currentTime = currentDate.getTime();
+    return Math.round((currentTime - this.startTime) / 1000);
+  }
+});
+
```

Comment:
Yes, but I think it's probably best to do in a separate PR. I'll add to my list for the sprint.